[2025-02-28T03:28:29.685+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file manual__2025-02-28T03:28:17.664074+00:00 [queued]>
[2025-02-28T03:28:29.699+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file manual__2025-02-28T03:28:17.664074+00:00 [queued]>
[2025-02-28T03:28:29.700+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-02-28T03:28:29.722+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_postgres_file> on 2025-02-28 03:28:17.664074+00:00
[2025-02-28T03:28:29.731+0000] {standard_task_runner.py:60} INFO - Started process 137 to run task
[2025-02-28T03:28:29.736+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl_dag', 'load_to_postgres_file', 'manual__2025-02-28T03:28:17.664074+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmpzjh1sm21']
[2025-02-28T03:28:29.740+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask load_to_postgres_file
[2025-02-28T03:28:29.810+0000] {task_command.py:423} INFO - Running <TaskInstance: crypto_etl_dag.load_to_postgres_file manual__2025-02-28T03:28:17.664074+00:00 [running]> on host 335ace54b661
[2025-02-28T03:28:29.912+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres_file' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T03:28:17.664074+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-28T03:28:17.664074+00:00'
[2025-02-28T03:28:29.916+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-02-28T03:28:29.919+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/scripts/load_to_postgres.py']
[2025-02-28T03:28:29.937+0000] {subprocess.py:86} INFO - Output:
[2025-02-28T03:28:31.814+0000] {subprocess.py:93} INFO - [INFO] Đã tải file '2025/02/28/10-21-14.csv' từ MinIO
[2025-02-28T03:28:31.815+0000] {subprocess.py:93} INFO - [INFO] Đã nạp 10 bản ghi vào Data Warehouse thành công.
[2025-02-28T03:28:31.816+0000] {subprocess.py:93} INFO - [INFO] Đã tải file '2025/02/28/10-28-26.csv' từ MinIO
[2025-02-28T03:28:31.817+0000] {subprocess.py:93} INFO - [INFO] Đã nạp 10 bản ghi vào Data Warehouse thành công.
[2025-02-28T03:28:31.817+0000] {subprocess.py:93} INFO - [INFO] Quá trình ETL hoàn tất. Đã xử lý 2 file mới từ MinIO.
[2025-02-28T03:28:31.933+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-02-28T03:28:31.966+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=crypto_etl_dag, task_id=load_to_postgres_file, execution_date=20250228T032817, start_date=20250228T032829, end_date=20250228T032831
[2025-02-28T03:28:32.004+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-02-28T03:28:32.044+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
